tests/unit/gapic/bigtable_admin_v2/test_bigtable_table_admin.py

```diff
@@ -18,6 +18,7 @@
 
 import grpc
 from grpc.experimental import aio
+import json
```

Comment:
Adding imports seems pointless (and lint-breaking) without also adding code which uses them.